### DISCUSSION TITLE
Installing xyce

### DISCUSSION THREAD

**Discussion by jarvis-lorry** [siliconcompiler/siliconcompiler#1013] [Help (Q&A)] [answered]
I've run the installation script for xyce included in the setup directory and while the build went just fine, I don't see where the executable for xyce resides.  There appears to be nothing installed called xyce or xycerun.  Was the build unsuccessful or do I need to use scripts from Sandia to get this to work?

**Comment by aolofsson** [comment, marked as answer]
Xyce is a pretty complex tool to install. You will need to follow their documentation EXACTLY to get it to work. Supporting tool installations for all the plug in tools supported by SC is unfortunately beyond what we can handle.

**Reply by jarvis-lorry** [reply]
Thank you.  I had hoped to avoid the pain of the xyce installation by using a complete script.

**Reply by mithro** [reply]
The [GitHub HDL organization](https://github.com/hdl) aims to provide prebuilt binaries for all the open source HDL / EDA tooling in [a number of different ecosystems](https://github.com/hdl/packages) (such as conda, containers, Termux, mingw, and others). It is already providing Surelog in a convenient manner which is used by Silicon Compiler.

It would be great to have Xyce as part of the [hdl/conda-eda](https://github.com/hdl/conda-eda) environment,  I'm somewhat surprised that it hasn't already been packaged and I have logged https://github.com/hdl/conda-eda/issues/210

There *are* containers for Xyce provided through [hdl/containers](https://github.com/hdl/containers) see the page @ https://hdl.github.io/containers/ToolsAndImages.html
